meta/group-info/zh.yaml

```diff
@@ -13,28 +12,28 @@ gmt:
   all_mods: 全局管理员
   areas:
     beatmap: 谱面管理员
-    chat: 聊天室管理员
+    chat: 聊天管理员
```

Comment:
`alumni.roles.chat` (line 6) also has the same term, so it should probably be updated as well

Reply:
fix it soon